src/engraving/libmscore/engravingitem.cpp

```diff
@@ -546,6 +551,11 @@ mu::draw::Color EngravingItem::curColor(bool isVisible, mu::draw::Color normalCo
     if (!isVisible) {
         return engravingConfiguration()->invisibleColor();
     }
+    if (engravingConfiguration()->isCurrentThemeHighContrast() && engravingConfiguration()->scoreInversionEnabled()
+        && !score()->isPaletteScore()) {
+        return mu::draw::Color(255, 255, 255, 160); //slightly dulled white for less strain on the eyes
```

Comment:
It would be better to return a full opacity off-white colour:

```c++
return mu::draw::Color(160, 160, 160, 255);
```

Alpha transparency affects performance because the program has to calculate the colour of pixels behind each object instead of just ignoring those pixels. Using a full opacity colour also prevents the colour changing when you have multiple semi-transparent objects on top of each other.

Reply:
Ahh I never would've thought of this!
Understood and fixed!